### ISSUE TITLE
[Feature Request]Medusa support

### ISSUE THREAD

**Issue by Merrick28** [Boerderij/Varken#172]
**Is your feature request related to a problem? Please describe.**
I tired to use sickchill backend to use Medusa, and it's not working.

**Describe the solution you'd like**
The Medusa api does not support the type "snatched"
The error message is 
```
{"data": {}, "message": "param: 'type' with given value ['missed', 'today', 'soon', 'later', 'snatched']  is out of allowed range ['later', 'missed', 'soon', 'today']", "result": "error"}
```

**Describe alternatives you've considered**
I would like to create another provider, first clone of sickchill, that removes the "snatched" type from the api. This provider could evolve if Medusa api differs later from sickchill api.

**Comment by samwiseg0** [comment]
Looking through their api documentation there does not apear to be any endpoints that will give what is missing, coming soon, being downloaded, or on today. Without any of those endpoints being documented we cannot develop for it. 

Can you provide the api document ion for the endpoints above?

**Comment by Merrick28** [comment]
No, nothing on their documentation... For now I only modified the varken/sicckchill.py file, and changed line 25 from 
```
params = {'cmd': 'future', 'paused': 1, 'type': 'missed|today|soon|later|snatched'}
```
to
```
params = {'cmd': 'future', 'paused': 1, 'type': 'missed|today|soon|later'}
```
to make it work, but I guess as it's not an official api, it may not last.

**Comment by samwiseg0** [comment]
Correct. As you pointed out they removed an endpoint from the API. Who knows what they will remove or add next. That would be the issue. Since there is nothing official from the developers we are at their mercy. In this state we have to reverse engineer the API and there is no telling if they will change it yet again. If the Medusa community can setup some documentation and have a stable API then we can develop for it.
